src/kepler_model/util/prom_types.py

```diff
@@ -72,6 +77,10 @@ def energy_component_to_query(component):
     return f"{node_query_prefix}_{component}_{node_query_suffix}"
 
 
+def vm_energy_component_to_query(component):
```

Comment:
I don't think we should use the the kepler_vm for training. We should use kepler_node but with job="metal" label (or job != "vm"). kepler_vm is not from the power meter but from the ratio which considering fairness distribution. In the case the we have only single vm, this might not be different but I think we better use the number from power meter.

Reply:
@rootfs thoughts? If I were to use kepler_node, I would need to find the process id for the virtual machine. If I recall, kepler_node with vm process id and kepler_vm are the same value or very similar.

Reply:
I don't think vm process will be the same if the idle power is not zero since it will distribute evenly for all running process not only the vm process but I think kepler_vm and kepler_node should be the same if you have a single VM.